ansible/playbooks/roles/prometheus-targets/tasks/main.yml

```diff
@@ -7,3 +7,12 @@
     mode: 0644
   with_fileglob:
   - file_sd/*.yml
+
+- name: Create discovery files from inventory
+  template:
+    src: "file_sd.yml.j2"
+    dest: "{{ prometheus_config_dir }}/file_sd/{{ item.name }}.yml"
```

Comment:
Are those files loaded by prometheus somewhere?

Reply:
Yes, this role was created before we had better support for file_sd in the cloudalchemy role. :smile: